### PR TITLE
Screenshot-able version of app (two tabs - one for each chart type)

### PR DESCRIPTION
**Pull request by fedderw**
The dropdowns in the bubble chart tab don't do anything yet, there is no callback written for it, but the default bubble chart already loads

@MaxGhenis @kvanalebeek

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -138,4 +138,7 @@ mysql*.png
 
 # dolt
 dolt_repo/
-.dolt/
\ No newline at end of file
+.dolt/
+
+# scratch
+*scratch*.*
```
Comment (MaxGhenis) [inline .gitignore:144]: add newline

File: py/visualize.py [hunk 2]
```diff
@@ -1,6 +1,9 @@
-import ubicenter
+# import ubicenter
 import plotly.express as px
+import plotly.graph_objects as go
 from py import preprocess_data as ppd
+import ubicenter
+import numpy as np
```
Comment (MaxGhenis) [inline py/visualize.py:7]: run black on this file

File: py/visualize.py [hunk 3]
```diff
@@ -51,33 +64,164 @@ def poll_vis(responses, poll_id, question_id=None, crosstab_variable="-"):
...
+    elif len(top_labels) == 3:
+        colors = [DARK_GRAY, GRAY, BLUE]
+    elif len(top_labels) == 2:
+        colors = [DARK_GRAY, BLUE]
     
-    target_responses["question_text_wrap"] = ppd.plotly_wrap(target_responses.question_text.copy(),130)
-    # question_text=target_responses["question_text_wrap"].iloc[0] 
-    question_text=target_responses["question_text_wrap"].unique()[0] if target_responses.shape[0] > 0 else "No question text"
-    
+    TEXTCOLORS = ['rgb(248, 248, 255)','rgb(248, 248, 255)','rgb(248, 248, 255)','rgb(248, 248, 255)','rgb(248, 248, 255)']
```
Comment (MaxGhenis) [inline py/visualize.py:118]:
```suggestion
    NEW_BLUE = 'rgb(248, 248, 255)'
    TEXT_COLORS = [NEW_BLUE] * 5
```
or something...please name the color descriptively (same goes for other `rgb` calls below) and include a comment as to why you need 5 of them

File: py/visualize.py [hunk 4]
```diff
@@ -51,33 +64,164 @@ def poll_vis(responses, poll_id, question_id=None, crosstab_variable="-"):
...
+
+    annotations = []
+
+    for yd, xd in zip(y_data, x_data):
+        # labeling the y-axis
+        annotations.append(dict(xref='paper', yref='y',
+                                x=0.14, y=yd,
+                                xanchor='right',
+                                text=str(yd),
+                                font=dict(family='Arial', size=14,
```
Comment (MaxGhenis) [inline py/visualize.py:171]:
```suggestion
                                font=dict(family='Roboto', size=14,
```
also below

**Review by MaxGhenis — COMMENTED** [review]
Thanks Will, could you drop some screenshots in a comment here too?

**Comment by fedderw** [comment]
> Thanks Will, could you drop some screenshots in a comment here too?

Sure thing. I forgot to push earlier, but I fixed the callback for the bubble chart so I'd wait until tomorrow to merge

**Comment by fedderw** [comment]
Here's bubble chart tab with cross tab selected:
image
Here's the bar chart tab:
image

**Comment by fedderw** [comment]
Here's an example of the bubble chart with no cross tab or bubble selected:
image

**Comment by fedderw** [comment]
If you just selected a poll from the default bubble chart, you get the no-crosstab option in the bar chart:
image

**Pull request merged**